### PR TITLE
feature/add_theme_custom_icons

### PR DESCRIPTION
**Pull request by feloxx**
add theme custion icons document.

---

Convenient Review address:

zh_cn: https://github.com/feloxx/vnote/blob/feature/add_theme_custom_icons/zh_cn/docs/%E7%94%A8%E6%88%B7/%E4%B8%BB%E9%A2%98%E5%92%8C%E6%A0%B7%E5%BC%8F.md

en_us: https://github.com/feloxx/vnote/blob/feature/add_theme_custom_icons/en_us/docs/Users/Themes%20and%20Styles.md

### CODE REVIEW

File: en_us/docs/Users/Themes and Styles.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Some key files of a theme:
 - `text-editor.theme`: theme file of the text editor (as well as Markdown editor);
 - `web.css`: style sheet file of the read mode of Markdown;
 - `highlight.css`: style sheet file of the read mode of Markdown for code block syntax highlight; VNote uses [Prism](https://prismjs.com/) for syntax highlight in read mode;
+- `icons`: customize the icon folder **optional** ；
```
Comment (tamlok) [inline en_us/docs/Users/Themes and Styles.md:20]:
```suggestion
- `icons`: folder for customized icons (**optional**);
```

File: zh_cn/blogs/file/vx.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "created_time": "2022-08-07T03:34:01Z",
+    "files": [
+        {
+            "attachment_folder": "",
+            "created_time": "2022-08-07T04:08:07Z",
+            "id": "61",
+            "modified_time": "2022-08-07T04:08:07Z",
+            "name": "法律.md",
```
Comment (tamlok) [inline zh_cn/blogs/file/vx.json:9]: Why

File: zh_cn/blogs/vx.json [hunk 3]
```diff
@@ -43,6 +43,9 @@
         }
     ],
     "folders": [
+        {
+            "name": "file"
```
Comment (tamlok) [inline zh_cn/blogs/vx.json:47]: Why

**Review by tamlok — CHANGES_REQUESTED** [review]

File: en_us/docs/Users/Themes and Styles.md (hunk 1 above)
Comment (feloxx) [reply en_us/docs/Users/Themes and Styles.md:20]: Done.

File: zh_cn/blogs/file/vx.json (hunk 2 above)
Comment (feloxx) [reply zh_cn/blogs/file/vx.json]: Done.

File: zh_cn/blogs/vx.json (hunk 3 above)
Comment (feloxx) [reply zh_cn/blogs/vx.json]: Done.

**Pull request merged**